### PR TITLE
Improve CHANGELOG.md

### PR DESCRIPTION
**Pull request by justin808**
* Add links to version diffs
* Add links to PRs and authors
* Add gem badge to README.md

### CODE REVIEW

**Comment by justin808** [comment]
Hi @grantspeelman Let's try to use PRs for all changes to master. How's this look? 😄

**Comment by justin808** [comment]
I added labels "review needed" and "approved for merge." Let's try to always "squash and merge" the PRs.

**Comment by justin808** [comment]
@grantspeelman Is the readme missing docs for the changes in the last few versions?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # CypressOnRails
 
-[![Build Status](https://travis-ci.org/grantspeelman/cypress-on-rails.svg?branch=master)](https://travis-ci.org/grantspeelman/cypress-on-rails)
+[![Build Status](https://travis-ci.org/grantspeelman/cypress-on-rails.svg?branch=master)](https://travis-ci.org/grantspeelman/cypress-on-rails) [![Gem Version](https://badge.fury.io/rb/cypress-on-rails.svg)](https://badge.fury.io/rb/cypress-on-rails)
```
Comment (grantspeelman) [inline README.md]: looks like we have to update the build status as well pointing to the new travis

**Comment by grantspeelman** [comment]
@justin808 not sure i understand what you mean by the Readme missing docs? can you please elaborate.

**Comment by justin808** [comment]
> @justin808 not sure i understand what you mean by the Readme missing docs? can you please elaborate.

I think I saw some features released without README updates (only changelog updates), but I may have been mistaken.

**Comment by justin808** [comment]
We can do the next changes on a separate PR.

**Pull request merged**